### DISCUSSION TITLE
Keep the data extracted from a replay visible without rewatching the play each time

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#19141] [Ideas] [answered]
Basically as stated above. Once the play has been watched once it should be feasible to store the information client side for a small quality of life increase right?

I'm surprised this one hasn't come up but I searched and didn't find any discussion

**Comment by integerrr** [comment, marked as answer]
- #14469
- #14472

**Reply by mk56-spn** [reply]
my bad
